### PR TITLE
More Mail to be had

### PR DESCRIPTION
**Pull request by BlueHNT**
## About The Pull Request
Added some more mail stuff.

## Why It's Good For The Game
It's fun to get something interesting and unique that ain't a god damn spam.

</details>

## Changelog
:cl:
tweak: tweaked job specific mails
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_mail.dm [hunk 1]
```diff
@@ -506,12 +518,18 @@
...
 		//NSV13
 		/obj/item/reagent_containers/food/drinks/drinkingglass/quadruple_sec = 8,
 		/obj/item/toy/figure/secofficer = 5,
 		/obj/item/reagent_containers/food/drinks/drinkingglass/quintuple_sec = 4,
+		// They're mocking you boy
+		/obj/item/clothing/under/syndicate/tacticool = 10,
+		// he will buy the SWAT pack anyway
+		/obj/item/combatgloves = 2,
+		// CHRISTMAS!?
+		/obj/item/gun/ballistic/shotgun/lever_action = 1,
```
Comment (bluewasabi24) [inline code/modules/jobs/job_mail.dm:532]: This is considered a hazard goodie, so please add this to the hazard goodie list in mail.dm so an admin log will be sent whenever it spawn.

**Comment by bluewasabi24** [comment]
Also, if you make any changes to the original code, you _need_ to notate it with `//NSV13`. Even if it's as small as changing the drop chance on something, you need to notate that so when we rebase, we know what to look for and update accordingly. 

The first example I see if you updating line **36** as `//BARTENDER` as opposed to the original `BART ENDER`
While minor, you need to add the `//NSV13` tag to the end of that as it is technically something you changed on code that's not originally ours. 

Also, not required but a polite ask, but if you could stay true to the descending order of drop chance for your new items, I'd really appreciate it. An example would be line 80 and 81 for your added Brig Physician drops. It just keeps it clean and easy to reference.

**Comment by BlueHNT** [comment]
> Also, if you make any changes to the original code (code specifically _not_ in the nsv13 folder), you _need_ to notate it with `//NSV13`. Even if it's as small as changing the drop chance on something, you need to notate that so when we rebase, we know what to look for and update accordingly.
> 
> The first example I see is you updating line **36** as `//BARTENDER` as opposed to the original `BART ENDER` While minor, you need to add the `//NSV13` tag to the end of that as it is technically something you changed on code that's not originally ours.
> 
> Also, not required but a polite ask, but if you could stay true to the descending order of drop chance for your new items, I'd really appreciate it. An example would be line 80 and 81 for your added Brig Physician drops. It just keeps it clean and easy to reference.

I have gone through some of the stuff and yes, I have noticed that I did put in some stuff incorrectly, I'm gonna revise it.

**Comment by BlueHNT** [comment]
> Power creep confirmed I'm denying these gamer items like atropine medipens, power tools, null rod and such

a bit yes, but I mean. Cargo techs straight up get a mosin. So it's kinda hard to draw a line

File: code/modules/jobs/job_mail.dm [hunk 2]
```diff
@@ -77,6 +77,8 @@
 
 		//NSV13
 		/obj/item/reagent_containers/food/drinks/drinkingglass/doctor_delight = 8,
+		/obj/item/reagent_containers/hypospray/medipen/stimpack = 2,
```
Comment (DuskLight9978) [inline code/modules/jobs/job_mail.dm:80]: This item is restricted to ERT, Antags, and admin. It is a highly powerful healing item. 10 u is enough to heal most spessmen a massive amount

File: code/modules/jobs/job_mail.dm [hunk 3]
```diff
@@ -346,6 +351,9 @@
 		/obj/item/toy/figure/hos = 5,
 		/obj/item/reagent_containers/food/drinks/drinkingglass/quintuple_sec = 4,
 		/obj/item/megaphone/command/rudiger = 4,
+		/obj/item/clothing/shoes/jackboots/fast = 2,
+		// You want it don't you?
+		/obj/item/melee/classic_baton/police/telescopic = 1,
```
Comment (DuskLight9978) [inline code/modules/jobs/job_mail.dm:356]: This powerful item is supposed to be restricted to heads of staff spawns

File: code/modules/jobs/job_mail.dm (hunk 2 above)
Comment (BlueHNT) [reply code/modules/jobs/job_mail.dm:80]: I'm in the middle of removing all the powercreep items, so give it time.

File: code/modules/jobs/job_mail.dm (hunk 3 above)
Comment (BlueHNT) [reply code/modules/jobs/job_mail.dm:356]: You already get one, just not telescopic, he also is a head and doesn't get one. Let him dream?

**Comment by BlueHNT** [comment]
> Power creep confirmed I'm denying these gamer items like atropine medipens, power tools, null rod and such

while I can understand power tools and null rod being a bit over the edge

You still get em quite easily along the round guaranteed, if you get 1 out of em earlier in the round, it won't ruin stuff, will it?

**Comment by BlueHNT** [comment]
This thing sucks.

**Pull request closed without merging**